src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -26,8 +26,11 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import com.google.common.base.Preconditions;
+
 import io.netty.buffer.ByteBuf;
 
+import net.minecraft.init.Items;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.init.Items'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)